### PR TITLE
cherry pick analytics issue

### PR DESCRIPTION
**Pull request by pinzart90**
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

(FILL ME IN) This section describes why this PR is here. Usually it would include a reference 
to the tracking task that it is part or all of the solution for.

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

(FILL ME IN) Brief description of the fix / enhancement. **Mandatory section** 

### Reviewers

(FILL ME IN) Reviewer 1  (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs [hunk 1]
```diff
@@ -64,4 +64,4 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyFileVersion("2.16.0.1640")]
```
Comment (mjkkirschner) [inline src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs:67]: is this branch based on 2.16 release? Is it expected the nuget version number does not match the build number - I can't recall.

File: tools/autobuild/build.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
   "product_id": "DYN",
   "release_id": "2.16.0",
```
Comment (mjkkirschner) [inline tools/autobuild/build.json:3]: release id does not change?

**Review by mjkkirschner — APPROVED** [review]
couple questions

File: LICENSE.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-DYNAMO v.2.15.0 © 2022 Autodesk, Inc.  All rights reserved.
+DYNAMO v.2.16.1 © 2022 Autodesk, Inc.  All rights reserved.
```
Comment (aparajit-pratap) [inline LICENSE.txt:1]: I thought this was bumped up already in the 2.16 release branch 😕

**Review by aparajit-pratap — APPROVED** [review]

File: LICENSE.txt (hunk 3 above)
Comment (mjkkirschner) [reply LICENSE.txt:1]: I did the about box content but must have missed this file - it doesn't make it into the application so it's not an issue for users IMO.

File: src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs (hunk 1 above)
Comment (pinzart90) [reply src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs:67]: The build number is based on a time dependent formula. I do not expect that to match between builds

Comment (mjkkirschner) [reply src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs:67]: yes, to be clear, I meant the `2.16.0.1640` was not the same as: 
https://www.nuget.org/packages/DynamoVisualProgramming.ZeroTouchLibrary/2.16.0.2501
perhaps the final build that was published to nuget was not the same build that was committed to git.

In any case, I want to be clear I am talking about the old build number, not the new one.

**Pull request merged**